src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileGenerator.cs

```diff
@@ -2533,6 +2582,59 @@ string GetBindingOptions()
 			}
 		}
 
+		private void BuildCustomMarkupExtensionPropertyValue(IIndentedStringBuilder writer, XamlMemberDefinition member, string prefix)
+		{
+			Func<string, string> formatLine = format => prefix + format + (prefix.HasValue() ? ";\r\n" : "");
```

Comment:
But this can be a local function :)

Reply:
It's now local in the method that uses it  :)